i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-drbd-ssh.md

```diff
@@ -53,33 +53,69 @@ doit avoir assez de privilèges pour executer la commande `/usr/sbin/drbdsetup`.
 
 ## Installation
 
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet 
+n'est pas requise pour voir apparaître le pack dans le menu **Configuration > Plugin Packs > Gestionnaire**.
+
+Au contraire, si la plateforme utilise une licence *offline*, installer le paquet 
+sur le **serveur central** via la commande correspondant au gestionnaire de paquet 
+associé à sa distribution: 
+
 <Tabs groupId="sync">
-<TabItem value="Online License" label="Online License">
+<TabItem value="RHEL, Oracle Linux, Alma Linux 8" label="RHEL, Oracle Linux, Alma Linux 8">
 
-1. Installer le Plugin sur tous les Collecteurs Centreon supervisant des ressources DRDB :
+```bash title='RHEL, Oracle Linux, Alma Linux version >= 8'
```

Comment:
Same comment as the previous one... And the tabs are enough, you don't need a title there.